packages/scripts/package.json

```diff
@@ -44,13 +44,15 @@
 		"command-exists": "^1.2.8",
 		"cross-spawn": "^5.1.0",
 		"decompress-zip": "^0.2.2",
+		"dir-glob": "^3.0.1",
 		"eslint": "^6.1.0",
 		"jest": "^24.7.1",
 		"jest-puppeteer": "^4.3.0",
 		"js-yaml": "^3.13.1",
 		"lodash": "^4.17.15",
 		"minimist": "^1.2.0",
 		"npm-package-json-lint": "^3.6.0",
+		"prettier": "npm:wp-prettier@^1.18.2",
```

Comment:
What functionality does the `npm:` prefix here perform if you don't mind, I've not seen this?

Reply:
It aliases `prettier` to `wp-prettier`, so it will actually use the `wp-prettier` package under the hood. This was introduced in npm 6.9.

Reply:
Neat, thanks, this is pretty cool

Reply:
We would need to enforce that `check-engines` requires npm 6.9 or higher by default.

Reply:
Even cooler would be not having to maintain such a fork, but progress is slow on that front: https://github.com/prettier/prettier/issues/5919

Reply:
Note that even if prettier/prettier#5919 was fixed and we could ship `wp-prettier` as a plugin, it wouldn't change much. We'd still need to maintain a fork of the JS formatter, and then ship the forked formatter as a plugin that replaces the original JS formatter. The code and the work is still the same, just shipped in different packaging.

I could fix prettier/prettier#5919 myself if needed, but with package aliasing support shipping both in NPM and Yarn, the aliasing method works equally well.

Reply:
> This was introduced in npm 6.9.

Yes, and it's the main reason why I waited so long before proposing to use the fork. Before package aliases, it could be distributed only as a tarball URL or a GitHub tag ref, which is kind of ugly and doesn't support versioning well. Now, distributing forks of things is much cleaner and easier.